### PR TITLE
Adds >memearrows to OOC

### PR DESCRIPTION
**Pull request by cumoverme**
greentext
>implying i know what i'm doing

### CODE REVIEW

**Comment by clusterfack** [comment]
What have you dooooooooone

**Comment by Intigracy** [comment]
I like you.

**Comment by PJB3005** [comment]
This might get used to get past admins setting OOC colour though

**Comment by ComicIronic** [comment]
Does this let you greentext the whole line from a > in the middle?

**Comment by Intigracy** [comment]
Afaik it only goes from the > if I'm reading the code right.

File: code/game/verbs/ooc.dm [hunk 1]
```diff
@@ -35,6 +35,9 @@
 			log_admin("[key_name(src)] has attempted to advertise in OOC: [msg]")
 			message_admins("[key_name_admin(src)] has attempted to advertise in OOC: [msg]")
 			return
+	var/msg_style = ""
+	if(findtext(trim(msg), "&gt;", 1, 5))
```
Comment (Intigracy) [inline code/game/verbs/ooc.dm]: >trim
Yeah this makes the entire message green.

**Comment by cumoverme** [comment]
@ComicIronic 
It only greentexts if the first character is a >.

**Comment by SkowronX** [comment]
Ogre the line

**Comment by PJB3005** [comment]
MERGE OR RIOT

**Comment by Exxion** [comment]
Riot it is, then

**Comment by SkowronX** [comment]
Aww

**Pull request closed without merging**